app/controllers/admin/two_factor_authentications_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Admin
+  class TwoFactorAuthenticationsController < BaseController
+    before_action :set_account
+
+    def destroy
+      @account.user.otp_required_for_login = false
```

Comment:
Can you do at least some and maybe all of:

- Roll all of this up into a method like `User#disable_two_factor!` or something (which would set required to false, clear the backups, and attempt save!)
- Add spec coverage for this controller action which shows that it does what it says
- Add model coverage for that method on User, if you create one like that